spi/src/main/java/org/eclipse/dataspaceconnector/spi/transfer/flow/DataOperatorRegistryImpl.java

```diff
@@ -0,0 +1,30 @@
+package org.eclipse.dataspaceconnector.spi.transfer.flow;
+
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+public class DataOperatorRegistryImpl implements DataOperatorRegistry {
```

Comment:
The implementation should go into the core:transfer module

Reply:
Moved accordingly